tests/unit_tests/test_math.py

```diff
@@ -205,6 +205,25 @@ def test_watt_spectrum():
     assert ref_val == test_val
 
 
+def test_normal_dist():
+    settings = openmc.capi.settings
+    settings.seed = 1
+    a = 14.08
+    b = 0.0
+    ref_val = 14.08
+    test_val = openmc.capi.math.normal_variate(a, b)
+
+    assert ref_val == test_val
```

Comment:
It looks like this is working fine, but let's error-proof ourselves just a bit with
```python
    assert ref_val == pytest.approx(test_val)
```

Reply:
resolved